### ISSUE TITLE
Linux support

### ISSUE THREAD

**Issue by Shished** [bradallred/CivVMacWorkshop#1]
Linux version of Civilization V is the same as Mac version, and workshop doesn't working for it.
Maybe this program will also work for Linux?

**Comment by bradallred** [comment]
Sadly, no it won't. Its written in Objective-C and doubtfully compatible with GNUStep. However, it is a very short program (less than 100 lines) so it would probably be very trivial for somebody familiar with Linux development to add a linux implementation.

The Steam API bits in the code should work on any platform.

**Comment by bradallred** [comment]
yes, looking at the code, most of it exists for dealing with things specific to Mac OS X so definitely not GNUStep compatible. However, it really is quite a trivial program: https://github.com/bradallred/CivVMacWorkshop/blob/master/CivVMacWorkshop/main.mm

even if you Aren't familiar with Objective-C it would be trivial to copy what its doing and do it in Linux.
